API/API.mli

```diff
@@ -373,19 +365,13 @@ sig
   val make_con : ModPath.t -> DirPath.t -> Label.t -> Constant.t
   [@@ocaml.deprecated "alias of API.Names.Constant.make3"]
 
-  val debug_pr_con : Constant.t -> Pp.std_ppcmds
 
-  val debug_pr_mind : MutInd.t -> Pp.std_ppcmds
 
   val pr_con : Constant.t -> Pp.std_ppcmds
 
-  val string_of_con : Constant.t -> string
 
-  val string_of_mind : MutInd.t -> string
```

Comment:
The same situation as with `Names.string_of_con`.

Reply:
Note that a few `to_string` and `print` in `Names` are "unsafe" (in the sense that they do not display a user-level name but only a debugging name). As far as I quickly checked, this is the case of `KerName.to_string`, `KerName.print`, `string_of_kn`, `MutInd.to_string`, `MutInd.print`, `string_of_mind`. I believe they should simply be removed from or renamed into a `debug` form (they are wrong e.g. for mutual inductive types, where the second component of a mutual inductive type use the first component in its kernel name).